docs/advanced.md

```
@@ -450,7 +468,7 @@ def process_inventory(df):
 
 If any rows fail validation, you'll get a detailed error message:
 
-```python
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)